src/components/service-metrics/metrics.tsx

```diff
@@ -12,6 +13,28 @@ interface ISummaries {
   readonly [key: string]: ReadonlyArray<IMetricSerieSummary>;
 }
 
+export function bytesLabel(value: number, _index: number): string {
+  const bytes = bytesConvert(value);
+
+  return `${bytes.value}${bytes.short}`;
+}
+
+export function percentLabel(value: number, _index: number): string {
+  return `${value}%`;
+}
+
+export function numberLabel(value: number, _index: number): string {
```

Comment:
I think we should expand the suffix options out here to cover billions

Reply:
Fair play. Updated.

You also made me realise that the graphs may potentially want to operate on `1.1`, `1.2`, `1.3` kinda thing. Added `.toFixed(2)` for good measure.